core/templates/dev/head/library/Library.js

```diff
@@ -92,7 +92,12 @@ oppia.controller('Library', [
       }
     };
 
+    var isCarouselScrolling = false;
```

Comment:
Just noticed something -- is this boolean meant to track a specific carousel, or all of them at once? If the latter, maybe rename to isAnyCarouselCurrentlyScrolling?